packages/frontend/src/ee/pages/AiAgents/AgentThreadPage.tsx

```diff
@@ -22,22 +10,7 @@ import {
 } from '../../features/aiCopilot/hooks/useOrganizationAiAgents';
 import { type AgentContext } from './AgentPage';
 
-type AiThreadMessageProps = {
-    message: AiAgentMessage;
-    agentName?: string;
-};
-
-const AiThreadMessage: FC<AiThreadMessageProps> = ({ message }) => {
-    const isUser = message.role === 'user';
-
-    return isUser ? (
-        <UserBubble message={message} />
-    ) : (
-        <AssistantBubble message={message} />
-    );
-};
-
-const AiAgentThreadPage = () => {
+const AiAgentThreadPage = ({ isPreview = false }: { isPreview?: boolean }) => {
```

Comment:
this `isPreview` is not used right? because we reuse the `AgentChatDisplay` instead of this component to display the conversation list in settings?

Reply:
yes, ty!